apps/dashboard/src/components/workflow-editor/add-step-menu.tsx

```diff
@@ -73,6 +75,7 @@ export const AddStepMenu = ({
   onMenuItemClick: (stepType: StepTypeEnum) => void;
 }) => {
   const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+  const areNewStepsEnabled = useFeatureFlag(FeatureFlagsKeysEnum.IS_ND_DELAY_DIGEST_EMAIL_ENABLED);
```

Comment:
Lol, the feature flag name is a bit off, but it's OK.

Reply:
this was already on ld